### PR TITLE
zanata-cli: new port, version 4.4.5

### PR DESCRIPTION
**Pull request by l2dy**

### CODE REVIEW

File: devel/zanata-cli/Portfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+PortSystem          1.0
+
+name                zanata-cli
+version             4.4.5
+categories          devel
+platforms           darwin
+license             LGPL-2.1+
+maintainers         {l2dy @l2dy} openmaintainer
+description         Zanata Java command-line client
+long_description    ${description}
```
Comment (pmetzger) [inline devel/zanata-cli/Portfile]: A better long description is probably in order. I think it's important that you can tell what something is, at least in a vague way, by reading the long description.

File: devel/zanata-cli/Portfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+checksums           rmd160  39638ec3f440249b001eeb7de8d3cbc318ae7c27 \
+                    sha256  9f7823b64c8feff17fb42a3925dba58563b102028ae883d104a89682106f95ba \
+                    size    16149108
+
+depends_run         bin:java:kaffe
+
+use_configure       no
+
+build {}
```
Comment (pmetzger) [inline devel/zanata-cli/Portfile]: Should this just be `build`? I can't remember, but I seem to recall that there's some such convention @ryandesign had been implementing across the Portfiles.

Comment (l2dy) [reply devel/zanata-cli/Portfile]: I don't think so, currently 227 ports have `^build {}$` and no port have `^build$`.

Comment (pmetzger) [reply devel/zanata-cli/Portfile]: I'm going to leave this be then, and if @ryandesign thinks it should be done differently I can fix it later or you can.

**Comment by pmetzger** [comment]
Merged! Thank you, @l2dy!

**Pull request merged**